### PR TITLE
Add UseExporter config option to AspNetCore example app

### PR DESCRIPTION
**Pull request by alanwest**
I noted that the AspNetCore example app had some commented out code that got missed in a few renaming refactors. For example, the rename of `AddHttpInstrumentation` -> `AddHttpClientInstrumentation`.

Adding a command line option `UseExporter` is one way to mitigate for this. Another simpler option could be to stay with the "uncomment this" approach and just have something like:

```
var exporter = "console";
// var exporter = "zipkin";
// var exporter = "jaeger";

if (exporter == "console") ...
else if (exporter == "zipkin") ...
...
```

Thoughts?

### CODE REVIEW

File: examples/AspNetCore/Program.cs [hunk 1]
```diff
@@ -30,7 +31,13 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
-                    webBuilder.UseStartup<Startup>();
+                    var config = new ConfigurationBuilder()
```
Comment (cijothomas) [inline examples/AspNetCore/Program.cs:34]: lets just stick the default config, and put this setting inside appsettings.json. I believe the current change breaks settings which are retrieved from appsettings.json now.

File: examples/AspNetCore/Startup.cs [hunk 2]
```diff
@@ -52,35 +52,39 @@ public void ConfigureServices(IServiceCollection services)
...
-                    o.ServiceName = this.Configuration.GetValue<string>("Zipkin:ServiceName");
-                    o.Endpoint = new Uri(this.Configuration.GetValue<string>("Zipkin:Endpoint"));
-                }));
-            */
-
-            services.AddOpenTelemetry((builder) => builder
-                .AddAspNetCoreInstrumentation()
-                .AddHttpClientInstrumentation()
-                .UseConsoleExporter());
+            // To choose which exporter to use, run this example with the UseExporter option (defaults to the ConsoleExporter):
```
Comment (cijothomas) [inline examples/AspNetCore/Startup.cs:55]: lets use appsettings.json for now - all other settings are configured there, so why not the choice of exporter as well.

File: examples/AspNetCore/Program.cs (hunk 1 above)
Comment (alanwest) [reply examples/AspNetCore/Program.cs]: Yep! You're completely right, apologies, I was making a hasty change.

**Comment by cijothomas** [comment]
Thanks!

**Pull request merged**